docs/explanations/quantization.md

```diff
@@ -15,19 +15,72 @@
  -->
 
 (quantization)=
-# Quantization
+# Quantization using Qwix
 
-Accurated Quantized Training is another technique that maps a subset of matrix multiplications in the training step to int8 to boost training efficiency.
+MaxText uses [Qwix](https://github.com/google/qwix), a JAX quantization library, to support various quantization techniques, including Quantization-Aware Training (QAT) and Post-Training Quantization (PTQ).
```

Comment:
Do we want to talk about both AQT and Qwix? Or are we ready to recommend only Qwix?